website/_dogfooding/clientModuleCSS.css

```diff
@@ -7,5 +7,5 @@
 
 /* Used to test CSS insertion order */
 .test-marker-site-client-module {
-  content: "site-client-module";
+  content: 'site-client-module';
```

Comment:
If you don't like the single quotes in the CSS, it could be overridden in the `.prettierrc`